Grow.c

```diff
@@ -3068,12 +3069,40 @@ static void catch_term(int sig)
 	sigterm = 1;
 }
 
+
+/**
+ * handle_forking() - Handle reshape forking.
+ *
+ * @forked: if already forked.
+ * @devname: device name.
+ * Returns: -1 if fork() failed,
+ *           0 if child process,
+ *           1 if job delegated to forked process or systemd.
+ *
+ * This function is a helper function for reshapes for fork handling.
+ */
+static int handle_forking(bool forked, char *devname)
+{
+	if (forked)
+		return 0;
```

Comment:
Probably you should add that if `forked` is set it assumes that we are in child process already, right?